wiki/Game_mode/osu!taiko/en.md

```diff
@@ -82,137 +82,16 @@ If [Relax](/wiki/Game_modifier/Relax) mod was used, the score judgement only acc
 
 ## Scoring
 
-*Scoring values can be found in [osu!taiko scoring system](/wiki/Score/osu!taiko).*
+[The osu!taiko score](/wiki/Gameplay/Score/osu!taiko) is a weighted sum of all hit objects' values. It depends on the following:
 
-Scoring section details all the intricacies of scoring, including mathematical formula.
+- [Judgement](/wiki/Gameplay/Judgement/osu!taiko) determines an object's base scoring value (300, 100, or 0 in case of a miss). The value of regular and large notes depends on hit timing; every other hit object's base value is constant.
+- [Accuracy](/wiki/Gameplay/Accuracy#osu!taiko) depends on judgement and shows how precise hits are. Late or early key presses, as well as misses, decrease overall accuracy.
+- [Combo](/wiki/Gameplay/Combo_(score_multiplier)) is a score multiplier: the higher it is, the more the object contributes to the total score. Combo may be [broken](/wiki/Glossary/Combobreak) by a miss. In osu!taiko, score multiplier derived from combo is limited and doesn't affect total score as much as it does in osu! or osu!catch.
+- [Kiai time](/wiki/Kiai_time): in osu!taiko, active kiai time increases score gain by 20%, just as it does in the original *Taiko no Tatsujin* series.
 
-The scoring terms in osu!taiko use the same terms used in *Taiko no Tatsujin* as shown below:
+In addition to score boost, each cleared object fills a small part of the [health bar](/wiki/Glossary/Health_bar), which must be at least **50% full** for a player to pass the beatmap.
```

Comment:
```suggestion
In addition to gaining score, each cleared object fills a small part of the [health bar](/wiki/Glossary/Health_bar), which must be at least **50% full** for a player to pass a beatmap.
```

Reply:
gaining -> giving? it's the player who gains

Reply:
hmm yeah you're right

Reply:
note that this works in context: "Score in osu!taiko is a weighted sum ... In addition to gaining score, each cleared object fills..." ("cleared object" is not the subject of the first clause; they're unrelated)